app/src/main/java/com/boolder/boolder/data/database/RoomDatabase.kt

```diff
@@ -25,3 +27,7 @@ abstract class BoolderAppDatabase : RoomDatabase() {
 }
 
 
+@Database(entities = [TickEntity::class], version = 3, exportSchema = true)
```

Comment:
As this will be the first time that the database will go in production, the database version should be kept to `1`. Otherwise it will already be mandatory to manage database migrations 😅

Reply:
Reset to 1 :)